src/controller/index.ts

```diff
@@ -250,13 +250,11 @@ export default class SimulariumController {
             return Promise.reject(e);
         }
 
-        return this.simulator
-            .convertTrajectory(dataToConvert, fileType)
-            .then(() => {
-                if (this.simulator) {
-                    this.simulator.requestSingleFrame(0);
-                }
-            });
+        return this.simulator.convertTrajectory(
+            dataToConvert,
+            fileType,
+            providedFileName
+        );
```

Comment:
i like this change. simpler, better decoupling

Reply:
re: switching files, I think so. Alli is looking into this an its one of the questions she is working on in regards to when we open and close new websockiet connections>

It's possible that if you open a new websocket connection when you open the different file, that the converted file would hit a websocket connection that is no longer active in the viewer and not load or cause an error. Typically the conversion happens so fast, and the menu to load other files is disabled during, so it's not something we've tested directly